tinycolor.js

```diff
@@ -13,7 +13,7 @@ var trimLeft = /^[\s,#]+/,
     mathMax = math.max,
     mathRandom = math.random;
 
-var tinycolor = function tinycolor (color, opts) {
+var tinycolor = function (color, opts) {
```

Comment:
What do you think about switching it to a function declaration only instead of the expression?  Seems like just `function tinycolor(color, opts) { ... }` would be good.